### PR TITLE
Implemented ssz_minimal_zero and MapObjectUtil object marshalling fro…

### PR DESCRIPTION
**Pull request by dangerousfood**
- [x] Implement ssz_minimal_zero

- [x] Implement MapObjectUtil object marshaling from YAML

### CODE REVIEW

**Review by jrhea — COMMENTED** [review]
Plz run spotless before merging

File: eth-reference-tests/src/test/java/pegasys/artemis/reference/TestSuite.java [hunk 1]
```diff
@@ -16,6 +16,10 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
 import com.google.errorprone.annotations.MustBeClosed;
+import kotlin.Pair;
```
Comment (jrhea) [inline eth-reference-tests/src/test/java/pegasys/artemis/reference/TestSuite.java:19]: what is this for?  curious

**Review by jrhea — APPROVED** [review]
spotless plz.  otherwise, lgtm

File: eth-reference-tests/src/test/java/pegasys/artemis/reference/TestSuite.java (hunk 1 above)
Comment (jrhea) [reply eth-reference-tests/src/test/java/pegasys/artemis/reference/TestSuite.java:19]: is it a tuple?

Comment (dangerousfood) [reply eth-reference-tests/src/test/java/pegasys/artemis/reference/TestSuite.java:19]: Yes, it is the kotlin library for Tuple which is JVM compatible.

**Pull request merged**